GVFS/GVFS/CommandLine/LogVerb.cs

```diff
@@ -60,6 +60,9 @@ public override void Execute()
 
                 string serviceLogsRoot = Paths.GetServiceLogsPath(this.ServiceName);
                 this.DisplayMostRecent(serviceLogsRoot, GVFSConstants.LogFileTypes.Service);
+
+                string autoUpgradeLogsRoot = Paths.GetServiceLogsPath(ProductUpgrader.UpgradeDirectoryName);
+                this.DisplayMostRecent(autoUpgradeLogsRoot, GVFSConstants.LogFileTypes.UpgradePrefix);

```

Comment:
@alameenshah  this is still producing misaligned outputs for the `gvfs log` command. Not a blocker here, but if we end up making other changes, it would be nice to clean this up as well since it looks unpolished.

Reply:
Added to the cleanup tasks.